environment.yml

```diff
@@ -2,13 +2,12 @@ name: confopt
 channels:
   - defaults
   - conda-forge
-  - psi4
+  - mcocdawc
```

Comment:
If we're using `pip` to install `chemcoord` we don't need this channel. He also said he'll delete it once the conda-forge package is merged.

Reply:
I fixed it.